platform/notifications/core/platform-core-notifications/src/main/java/org/codice/ddf/notifications/impl/FileSystemPersistenceProvider.java

```diff
@@ -104,14 +105,15 @@ public void store(String key, Object value) {
                 }
             }
             file = new FileOutputStream(getMapStorePath() + key + PERSISTED_FILE_SUFFIX);
-            OutputStream buffer = new BufferedOutputStream(file);
+            buffer = new BufferedOutputStream(file);
             output = new ObjectOutputStream(buffer);
             output.writeObject(value);
         } catch (IOException e) {
             LOGGER.info("IOException storing value in cache with key = {}", key, e);
         } finally {
             IOUtils.closeQuietly(output);
             IOUtils.closeQuietly(file);
+            IOUtils.closeQuietly(buffer);
```

Comment:
Same comment about finally.

Reply:
Updated to use try with resource in each of these files.